test/corpus/expressions.txt

```diff
@@ -1474,56 +1474,122 @@ i = <Foo:Bar bar={}>{...children}</Foo:Bar>
 ---
 
 (program
-  (variable_declaration (variable_declarator (identifier)
-    (jsx_element (jsx_opening_element (identifier)) (jsx_closing_element (identifier)))))
-  (expression_statement (assignment_expression (identifier)
-    (jsx_element
-      (jsx_opening_element (member_expression (identifier) (property_identifier)))
-      (jsx_closing_element (member_expression (identifier) (property_identifier))))))
-  (expression_statement (assignment_expression (identifier)
-    (jsx_element (jsx_opening_element) (jsx_self_closing_element (identifier)) (jsx_closing_element))))
-  (expression_statement (assignment_expression (identifier)
-    (jsx_element
-      (jsx_opening_element (identifier))
-      (jsx_self_closing_element (identifier))
-      (jsx_closing_element (identifier)))))
-  (expression_statement (assignment_expression (identifier)
-    (jsx_self_closing_element (identifier) (jsx_attribute (property_identifier)))))
-  (expression_statement (assignment_expression (identifier)
-    (jsx_self_closing_element
-      (identifier)
-      (jsx_attribute (property_identifier) (string (string_fragment)))
-      (jsx_attribute (property_identifier) (jsx_expression (number)))
-      (jsx_attribute (property_identifier) (string (string_fragment)))
-      (jsx_attribute (property_identifier)))))
-  (expression_statement (assignment_expression (identifier)
-    (jsx_self_closing_element
-      (identifier)
-      (jsx_attribute
-        (property_identifier)
-        (jsx_expression (member_expression (identifier) (property_identifier)))))))
-  (expression_statement (assignment_expression (identifier)
-    (jsx_element
-      (jsx_opening_element
-        (identifier)
-        (jsx_attribute
+  (variable_declaration
+    (variable_declarator
+      name: (identifier)
+      value: (jsx_element
+        open_tag: (jsx_opening_element
+          name: (identifier))
+        close_tag: (jsx_closing_element
+          name: (identifier)))))
+  (expression_statement
+    (assignment_expression
+      left: (identifier)
+      right: (jsx_element
+        open_tag: (jsx_opening_element
+          name: (member_expression
+            object: (identifier)
```

Comment:
I saw in the docs that you can specify field names in tests so added these here to actually test this change

Then the test was failing unless _all_ field names were specified so replaced the existing test output with what it was now emitting (I assume it's all "correct" including all of the newly-added field names here?)

Reply:
yeah that's normal - tests w/ fields need all fields added

Reply:
https://github.com/github/docs/pull/21615#issue-1423100071